Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿// This is the Projector Abstract class
+// Abstract means the class can only be accessed through a child class. 
+// Abstract methods, can only be used by an abstract class, doesn't have a body, only provided from inheritence 
+// This class will contain all common RS 232 commands across all projectors
+
+// The abstract class's methods can only be accesed through inheritence 
+
+using System;
+
+namespace RS232_API
+{
+    abstract class Program
+    {
+        protected String Make;
+        protected String Model; 
+        public abstract void ON();
+        public abstract void OFF();
+        public abstract void Generation();
+        public abstract void Name();
+
+        
+    }
+    // Members marked as abstrac must be implemented by non-abstract classes that derive from the abstract class 
+    class Panasonic : Program

```

Comment:
This shouldn't be here at all, there will be another class per model which we build afterwards